src/components/timecard/edit-shift-timecard/EditShiftTimecard.jsx

```diff
@@ -14,19 +14,25 @@ import { validateServiceErrors } from '../../../utils/api-utils/ApiUtils';
 import { useApplicationContext } from '../../../context/ApplicationContext';
 
 const EditShiftTimecard = ({ timeEntry, timeEntriesIndex }) => {
-  const toggleEditShiftHours = (event) => {
-    event.preventDefault();
-    setShowEditShiftHours(!showEditShiftHours);
-  };
-
   const { setServiceError } = useApplicationContext();
   const { timeEntries, setTimeEntries } = useTimecardContext();
 
-  const timeEntryExists = !!timeEntry?.startTime;
+  const timeEntryExists = !!timeEntry?.startTime && timeEntry?.startTime !== '';
```

Comment:
nit: I don't think you need the second question mark `timeEntry?` as you know you have the object from the first condition.
Think it can be: `...&& timeEntry.startTime !== ''`

Reply:
Done